contributing.md

```diff
@@ -383,10 +384,10 @@ Merging strategy:
 
 * For internal contributions, give the original author some time to hit the merge button themselves / check directly with them if it’s ok to merge.
 * Default to “squash and merge”
-  * Make sure the commit message captures the core ideas of the pull request.
-* “Rebase and merge” when moving files (do a `git mv` as a separate commit)
-* “Create a merge commit” when porting changes across branches
-* Refrain from force-pushing while the PR is under review (which includes rebasing and squashing)
+  * Make sure the commit message captures the core ideas of the pull request and contains the author's signature.
+* “Rebase and merge” when moving files (do a `git mv` as a separate commit).
+* “Create a merge commit” when porting changes forward and "Rebase and merge" when porting backwards.
```

Comment:
```suggestion
* Create a "merge commit” when porting changes forward, and "Rebase and merge" when porting backwards.
```

Reply:
The button says "Create a merge commit" though :wink: 

![image](https://user-images.githubusercontent.com/5751272/96156798-58bc9700-0ec6-11eb-9b6e-55d9176c8706.png)